index.js

```diff
@@ -183,6 +183,6 @@ module.exports.expandInputs = expandInputs;
 module.exports.formats = {
   html: require('./lib/output/html'),
   md: require('./lib/output/markdown'),
-  mdast: require('./lib/output/markdown_ast'),
+  remark: require('./lib/output/markdown_ast'),
```

Comment:
If the AST format defined by remark is still being called "mdast", this should still be "mdast" too, right?